### ISSUE TITLE
Add the CURL run button on the Client page TEST

### ISSUE THREAD

**Issue by MichaelRSilva** [bancodobrasil/mtls-best-friend#8]
On the `/check-client` page, exemplify the CURL command. An improvement would be to add a `RUN` button that already executed this command on the screen and showed the result.

**Comment by hasanosman2020** [comment]
Hi, I am interested in assisting you but I'm relatively new to the field and do not fully understand your issue. I will of course look it up online but I may need some guidance. However, if you require a helper already familiar with the issue then I wish you all the best.

**Comment by tiagostutz** [comment]
Hi @hasanosman2020 👋🏻 ! Welcome.!

Let's try to tackle this.! Are you new to mTLS field or NextJS? If you are able to run the project locally, that's a good start. Have you already done that?

The details about mTLS and stuff we can guide you throughtout the issue development.

**Comment by hasanosman2020** [comment]
Hi @tiagostutz. I am new to both mTLS and NextJS. I haven't worked with either before but. have googled them and am confident that I can get to grips with both. I encountered a few issues when trying to run the project but I think that I have managed to debug them. I have taken a screenshot of the project after running yarn dev - is that what is supposed to show on screen? 
<img width="1403" alt="yarndevresult" src="https://user-images.githubusercontent.com/7830170/112761866-22e31900-8ff5-11eb-9825-5d48ea0d62b8.png">
<img width="1403" alt="yarndevresult" src="https://user-images.githubusercontent.com/7830170/112761904-57ef6b80-8ff5-11eb-8fcd-f3bfbbd09a7a.png">



Thanks
Hasan

**Comment by tiagostutz** [comment]
Right, I have made the full path here cloning this repo from scratch, running `yarn install` and `yarn run dev` from the **app** folder and it worked. So we have to dig a little deep in your scenario to find out whats happening.

1. What NodeJS version are you using (`run node -v in the same terminal you are running `yarn run dev`)
1. Have you cloned from your fork or from this original repo (we recommend cloning from your fork)
1. Double check if you ran `yarn install` at the **app** folder and if the node_modules folder are there
1. Those CSS files pointed as missing are indeep in the folder?

**Comment by hasanosman2020** [comment]
Hi @tiagostutz - I believe that I have the project running locally. I did what you suggested - I forked the repo then cloned from my fork and I installed and ran yarn dev from the app folder. Thanks for your help. I have attached a screenshot of running yarn dev.  So I'm good to go.
<img width="1416" alt="mTLSBestFriend" src="https://user-images.githubusercontent.com/7830170/112838085-98ea8d00-9094-11eb-95fe-a6a2d58ab815.png">

**Comment by tiagostutz** [comment]
Hi @hasanosman2020 are you still interested in tackling this issue?

**Comment by reinaldocomputer** [comment]
Hello,

Please check it out my PR and see if it solve this problem:

https://github.com/bancodobrasil/mtls-best-friend/pull/13

Thanks
